lib/DependencyLoader.js

```diff
@@ -112,6 +112,13 @@ class DependencyLoader {
         })
       }
     })
+    Object.entries(this.configs).forEach(([modName, config]) => {
+      if (config.alias) {
+        config.name = config.alias
```

Comment:
Is removing the original name a necessary step? (As we're still registering the instance with the alias name, which from memory is the only bit that matters) 

Think it would be useful to keep the original name for transparency - e.g. the logs would show the non-alias name

---

I think we should also log the override, not sure which level makes sense - possibly a warning? (Which would make sense given the function you added, see below comment)

Reply:
Ah yes, damn. Let's maybe forget that bit then, as it has the possibility of being a bit confusing anyway.

Let's just log the override message. Don't mind what log level you use; debug, warning and info all seem reasonable.